packages/react-ui-kit/src/Layout/Theme.tsx

```diff
@@ -142,6 +144,7 @@ export const themes: {[themeId in THEME_ID]: Theme} = {
       disabledColor: COLOR_V2.GRAY_60,
       contrastTextColor: COLOR.BLACK,
       borderColor: COLOR_V2.GRAY_90,
+      descriptionColor: COLOR_V2.WHITE,
```

Comment:
just wanted to confirm that descriptionColor is white(descriptionColor: COLOR_V2.WHITE) for both light and dark theme?
@przemvs